pkg/kv/kvclient/kvcoord/dist_sender_rangefeed.go

```diff
@@ -262,8 +263,9 @@ func (ds *DistSender) partialRangeFeed(
 	// Register partial range feed with registry.
 	active := &activeRangeFeed{
 		PartialRangeFeed: PartialRangeFeed{
-			Span:    span,
-			StartTS: startFrom,
+			Span:            span,
+			StartTS:         startFrom,
+			EstablishedTime: timeutil.Now(),
```

Comment:
I'm not sure what exactly we want the semantics of this field to be, but maybe we should set it whenever the rangefeed is actually established, i.e. when we get a non-`nil` error back from `client.RangeFeed()`? At least to me, "established" implies that the rangefeed was successfully set up. I guess that also implies that it would get updated if we ever reconnect too.

Alternatively, we could consider calling this `Created` and use `Established` as a separate field.